<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>490</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>710</width>
    <height>490</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>710</width>
    <height>490</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>DivX Converter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="FPSlabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Current FPS:</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="FPSDoubleSpinBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>240.000000000000000</double>
    </property>
    <property name="value">
     <double>25.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="iFPSlabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Loading FPS:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="iFPSDoubleSpinBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>240.000000000000000</double>
    </property>
    <property name="value">
     <double>25.000000000000000</double>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchLineEdit">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>691</width>
      <height>401</height>
     </rect>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Page</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="ConsLabel">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Subtitle consistent</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>710</width>
     <height>21</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>true</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuSave_File_As">
     <property name="title">
      <string>Save File As</string>
     </property>
     <addaction name="actionSaveAsSRT"/>
     <addaction name="actionSaveAsMPSub"/>
     <addaction name="actionSaveAsMicroDVD"/>
    </widget>
    <addaction name="actionNew_File"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_File"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_File"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="menuSave_File_As"/>
    <addaction name="separator"/>
    <addaction name="actionClose_File"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionAdd_Subtitle"/>
    <addaction name="actionEdit_Subtitle"/>
    <addaction name="actionDelete_Subtitle"/>
   </widget>
   <widget class="QMenu" name="menuAdvanced">
    <property name="title">
     <string>Advanced</string>
    </property>
    <widget class="QMenu" name="menuTime_Shift">
     <property name="title">
      <string>Time Shift</string>
     </property>
     <addaction name="actionShift_Sellection"/>
     <addaction name="actionShift_All"/>
    </widget>
    <addaction name="menuTime_Shift"/>
    <addaction name="actionAuto_Concat"/>
    <addaction name="actionAuto_Devide"/>
    <addaction name="actionConcat_Files"/>
    <addaction name="actionSplit_File"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAdvanced"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveAsSRT">
   <property name="text">
    <string>SRT</string>
   </property>
  </action>
  <action name="actionSaveAsMPSub">
   <property name="text">
    <string>MPSub</string>
   </property>
  </action>
  <action name="actionSaveAsMicroDVD">
   <property name="text">
    <string>MicroDVD</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionClose_File">
   <property name="text">
    <string>Close File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionEdit_Subtitle">
   <property name="text">
    <string>Edit Subtitle</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionAdd_Subtitle">
   <property name="text">
    <string>Add Subtitle</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionDelete_Subtitle">
   <property name="text">
    <string>Delete Subtitle</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionNew_File">
   <property name="text">
    <string>New File</string>
   </property>
  </action>
  <action name="actionShift_Sellection">
   <property name="text">
    <string>Shift Sellection</string>
   </property>
  </action>
  <action name="actionShift_All">
   <property name="text">
    <string>Shift All</string>
   </property>
  </action>
  <action name="actionAuto_Concat">
   <property name="text">
    <string>Auto Concat</string>
   </property>
  </action>
  <action name="actionAuto_Devide">
   <property name="text">
    <string>Auto Devide</string>
   </property>
  </action>
  <action name="actionConcat_Files">
   <property name="text">
    <string>Concat Files</string>
   </property>
  </action>
  <action name="actionAbout_File">
   <property name="text">
    <string>About File</string>
   </property>
  </action>
  <action name="actionSplit_File">
   <property name="text">
    <string>Split File</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
